Amazon.IonObjectMapper/IonObjectSerializer.cs

```diff
@@ -173,11 +170,6 @@ public override void Serialize(IIonWriter writer, object item)
                     // This Ion property name was already serialized.
                     continue;
                 }
-                
-                if (property.GetCustomAttributes(true).Any(it => it is IonIgnore))
-                {
-                    continue;
-                }
```

Comment:
Why are we now serializing things with IonIgnore attribte?

Reply:
This check was removed because it was made redundant by the change made to `GetValidProperties`.